airbyte-integrations/connectors/source-db2/src/main/java/io.airbyte.integrations.source.db2/Db2Source.java

```diff
@@ -20,7 +22,7 @@
   public static final String DRIVER_CLASS = "com.ibm.db2.jcc.DB2Driver";
 
   public Db2Source() {
-    super(DRIVER_CLASS, new Db2JdbcStreamingQueryConfiguration());
+    super(DRIVER_CLASS, new Db2JdbcStreamingQueryConfiguration(), new Db2SourceOperations());
```

Comment:
I guess you might use getSourceOperations() instead of declaring new Db2SourceOperations

Reply:
Unfortunately, I cannot do that. This method cannot be used here. This method must be static in order to apply it in the constructor

Reply:
I took it out into a separate method and designed it as a singleton